### DISCUSSION TITLE
Mobilizon (eventbright, facebook events, meetup alternative)

### DISCUSSION THREAD

**Discussion by RayBB** [coollabsio/coolify#2222] [Service Template Requests]
This is an awesome federated alternative for planning events. Would be great to see it on coolify!

Docker Install Instructions: https://docs.joinmobilizon.org/administration/install/docker/
Homepage: https://mobilizon.org/en/
Wikipedia: https://en.wikipedia.org/wiki/Mobilizon

**Comment by sxiii** [comment]
I have semi-successfully deployed Mobilizon but it performs really weirdly. It seems to be working but REEALLY slow with loading events, also it does not correctly load historical (passed) events.

I've created a separate bug for it in the issues: https://github.com/coollabsio/coolify/issues/6184
